aegis_bringup/README.md

```diff
@@ -36,3 +36,109 @@ effort: [0.0, 0.0]"
 ```bash
 ros2 service call /dashboard_client/play std_srvs/srv/Trigger {}
 ```
+
+## Launch files diagram
+
+TODO - embed dynamic PlantUML generation:
+```markdown
+![alternative text](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.github.com/plantuml/plantuml-server/master/src/main/webapp/resource/test2diagrams.txt
+```
+
+```plantuml
+@startuml Launch Files
+
+package aegis_bringup {
+    class bringup << (L,#FF7700) LaunchFile >> {
+        Arg namespace=""
+        Arg mock_hardware="false"
```

Comment:
Update the diagram with the rest of the launch args (i.e. `launch_rviz` and `tf_prefix`)